### ISSUE TITLE
cTIVO remains in "Paused" state and will no longer encode

### ISSUE THREAD

**Issue by themuzikcafe** [mackworth/cTiVo#501]
First and foremost, I would like to thank you for a GREAT application. I've used it for several years!

Recently I had to replace an external drive on my (original) Tivo Bolt. Things were fine for a week or So, but recently, When cTivo goes to encode something, either from a show subscription or I choose manual download, it will either sit in the ready state or after 1 or 2 programs being encoded, it will automatically go into pause mode. Restarting my MAC has temporarily solved this, and it will encode again, but after 1 or 2 shows it goes into pause mode again.

I'm running the latest version of MACOS (updated last week).

I do not have any of the preferences checked to pause it on a schedule. I have refreshed the subscriptions and have deleted them altogether.

I kind of feel like its a problem with the Bolt communicating with the cTIVO software. I have restarted the router on my network as well as restarted the Bolt several times.

I'm a bit puzzled as to what has gone on. I'm usually pretty good at diagnosing tech problems (been doing it for 27 years as my job LOL)

Any help you could offer would be greatly appreciated. I know your time is extremely valuable and I want to respect that.

Thank you in advance for your help and continued efforts.

EB

**Comment by mackworth** [comment]
Thank you for the nice compliment!

couple questions: By Paused Mode, you mean the big word Paused appears, or just that nothing happens? What do you mean by “Restarting my MAC”?  Latest MacOS version, I assume you don’t mean beta of next version. Was update OS before/after drive and/or problem starting?

To diagnose, can you set logging mode to Major, and run through restarting it, doing some shows until it pauses, then use Help>show in finder, cmd-click to compress the log directory and then post it here (don’t email it, as GitHub doesn’t pass the attachments through).

**Comment by themuzikcafe** [comment]
Good morning ... Thank you for getting back to me. I'm not near my machine at the moment but will send you the log files as per your instructions as soon as I return to it.

When adding a show to the cue, The big red word Paused appears and the application does nothing. 

I close the application, restart my mac, open cTIVO back up and it encodes 1 or 2 shows, but then pauses and the word appears over the Cue again, and the application stops functioning. 

MacOS version is 12.6 (apologies for not including that in my initial post) The update to macOS was after I replaced/reformatted the drive. A few more notes on an external drive. I was getting sporadic reboots on my Bolt, and after reading a bit on some forums I gathered this may be an indication of a bad drive, so I started by replacing the external drive. The replacement proved to not work with the BOLT, so I simply reformatted the original external drive. And that is currently working for several weeks without issue.

Several days ago, I started seeing this pause issue with cTIVO. Quiet bizarre as your application has run flawlessly for me for years.

I deleted cTIVO off my MAC completely and reinstalled it (Version 3.5.2 (1139)) several hours ago, I'm currently encoding 2 or 3 shows and will get you the log files as soon as I return to the machine.

Thank you so much for your valuable time

**Comment by mackworth** [comment]
Haven’t checked code yet, but one thought is to confirm where your download directory (main screen) and temp directory (Adv prefs) are pointing to, especially if to old drive name or out of space. That would cause a Pause. It should warn you if there’s any problem, but one never knows!

**Comment by themuzikcafe** [comment]
Good afternoon,

I finally made it back to my machine in my home office. Please find attached the log files you requested.

I also think that maybe you were onto something there with your earlier suggestion, so I looked at the space left on the drive. I had only about 50 or 150GB I think left. So indeed that may have been the cause of the entire pause problem as the cTIVO when encoding didn't have any room for temp or finished files. I'm not sure the attached logs will show any of the paused errors, as the three shows I tested after making room on the drive seem to work fine.

cTiVo.zip

I will keep an eye on it for the next few days/hours and let you know how it's doing.

I truly appreciate your valuable time!

EB

**Comment by themuzikcafe** [comment]
[cTiVo.zip](https://github.com/mackworth/cTiVo/files/9659607/cTiVo.zip)
 Sorry, here are the logs

**Comment by mackworth** [comment]
Yup, looks like that’s the reason:
```
2022-09-27 09:26:08:925 -[MTDownload configureFiles]@741>Disk space problem: 3856MB tmp and 3856MB download vs 5410MB fileSize
2022-09-27 09:26:08:925 -[MTTiVoManager notifyForName:withTitle:subTitle:isSticky:]@1747>Notify: Kevin Can **** Himself: The Machine: Pausing downloads: Your download disk is low on space
Probably need to delete some files.
```

So it’s only seeing 3GB before trying to download a 5GB file. It tried to warn you, but I take it you didn’t see that msg. I’ll have to check on warnings with latest OS.

**Comment by themuzikcafe** [comment]
Thank you SO MUCH! Truly appreciated!!! Looks like its time for a bigger HD :) 

Thank you so much again! Have a great night!

**Comment by themuzikcafe** [comment]
Hi Hugh,

I hope things are well. 

I recently updated my Tivo from a Bolt to a Edge.

I set the Tivo Edge up, went over to my Mac and it found the new TIVO inside cTIVO immediately. I didn't change anything.

I noticed that when the cTivo encodes the new recordings from the Edge, the video is pixelated and all blurry. Is there something I need to do as far as encoding? Does the Tivo Edge require a different encoder or setting within cTivo?

Thank you so MUCH for this awesome product!

I look forward to hearing from you soon. Thank you in advance.

EB
